build.gradle.kts

```diff
@@ -148,19 +148,19 @@ kotlin {
             }
         }
 
-        compilations["main"].packageJson {
+        (compilations["main"] as org.jetbrains.kotlin.gradle.plugin.mpp.KotlinJsCompilation).packageJson {
             main = "kotlin/kotlinx-html-js.js"
             name = "kotlinx-html-js"
         }
 
-        compilations["main"].kotlinOptions.apply {
+        (compilations["main"] as org.jetbrains.kotlin.gradle.plugin.mpp.KotlinJsCompilation).kotlinOptions.apply {
```

Comment:
Why do we need to cast it here in M1?

Reply:
It doesn't work without cast

Reply:
@ilgonmic

Reply:
Honestly, I fixed it and backproted to 1.4
https://github.com/JetBrains/kotlin/blob/1.4.0/libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/targets/js/dsl/KotlinJsTargetDsl.kt#L58
So it can be omitted, I guess

Reply:
Reverted